Gems/ScriptCanvas/Code/Include/ScriptCanvas/Translation/GraphToLua.cpp

```diff
@@ -962,7 +1045,10 @@ namespace ScriptCanvas
                 for (auto& out : nodeAndParse->m_latents)
                 {
                     m_dotLua.WriteNewLine();
-                    TranslateNodeableOut(nodeAndParse->m_nodeable, out.second);
+                    TranslateExecutionOut
+                        ( nodeAndParse->m_nodeable
+                        , out.second
+                        , nodeAndParse->m_isInterpreted ? OutSourceType::InterpretedClass : OutSourceType::Nodeable);
```

Comment:
Question: what is the difference between interpretedclass and nodeable here?

Reply:
Nodeables are C++ classes that are derived from Nodeable. InterpretedClasses are SC graphs that during translation will be implemented as classes (whether as "classes" in Lua, or compiled to a new type in C++).